package.json

```diff
@@ -11,10 +14,24 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint $npm_package_path_client && stylelint $npm_package_path_client",
```

Comment:
These `lint` scripts and  `precommit` functions may be managed more properly.

Reply:
I think running `eslint` and `stylelint` in this `lint` command looks fine to me 👀